### PR TITLE
Updated methoden page

### PR DESCRIPTION
**Pull request by onursagir**
- [x] Added icon and href attributes to the methoden datatype in the cms.
- [x] Updated the methoden view including stories.

todo:
- [x] Determine and implement max width for container (grid). The container currently does not have a max width making things look funky at a larger sizes. (@Henneddd & @onursagir)
- [ ] Determine if we want to have masonry or grid layout for methoden page.  (@Henneddd & @onursagir)
- [ ] After merge update methoden to have icon and href. (@sgort)

### CODE REVIEW

File: apps/web/src/services/cms/mocks/get-methods-mock.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        createdAt: '2022-12-08T12:32:57.293Z',
+        updatedAt: '2023-01-20T08:38:36.872Z',
+        publishedAt: '2022-12-08T12:34:12.614Z',
+        locale: 'en',
+        Href: '#',
+        Icon: 'right-angle-ruler',
+      },
+    },
+    {
+      id: 2,
```
Comment (sgort) [inline apps/web/src/services/cms/mocks/get-methods-mock.ts:48]: Staat deze er nu per ongeluk 2x in?

**Review by sgort — CHANGES_REQUESTED** [review]
Check de comment over dubbel voorkomen FLINT tekst

**Review by Henneddd — CHANGES_REQUESTED** [review]
Added documentation providing details such as max height, width, text height.
Additionally keep using grid system, CTRL + G in Figma to show.

**Review by Henneddd — APPROVED** [review]

**Comment by onursagir** [comment]
>  Determine if we want to have masonry or grid layout for methoden page. (@Henneddd & @onursagir)

Will be discussed during the wednesday meeting

File: apps/web/src/services/cms/mocks/get-methods-mock.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        createdAt: '2022-12-08T12:32:57.293Z',
+        updatedAt: '2023-01-20T08:38:36.872Z',
+        publishedAt: '2022-12-08T12:34:12.614Z',
+        locale: 'en',
+        Href: '#',
+        Icon: 'right-angle-ruler',
+      },
+    },
+    {
+      id: 3,
```
Comment (sgort) [inline apps/web/src/services/cms/mocks/get-methods-mock.ts:48]: Zie de FLINT nog steeds dubbel. 1x met Id 2 en hier met Id 3

**Review by sgort — CHANGES_REQUESTED** [review]
Zie comment over dubbele FLINT

**Review by sgort — APPROVED** [review]

**Pull request merged**